lib/packwerk/application_validator.rb

```diff
@@ -207,7 +207,7 @@ def check_inflection_file
     end
 
     def check_acyclic_graph
-      packages = Packwerk::PackageSet.load_all_from(".")
+      packages = Packwerk::PackageSet.load_all_from(@configuration.root_path)
```

Comment:
We need this fix if we want to use temporary application directories in our tests. See https://github.com/Shopify/packwerk/pull/59

Reply:
Nice! One small thing, can you move the bug fix into a separate commit on this PR? That way we can retain more context on why changes were made and it becomes easier to review the PR.

This is a good talk that explains it in more detail http://yt.dudzik.co/watch/?v=qpdYRPL3SVE